env/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
This crate is essentially a place where we dump stuff that we don't know where else to put. I don't think root of nearcore is a good place for it and I think it should have a more descriptive name.

In general, instead of extracting actix from the primitives crate we should be extracting stuff used by the runtime into finer grained crates, leaving everything else behind in the primitives crate. It requires revisiting every type from primitives used by the runtime and deciding which category of primitives it better represents. It is a much harder thing to do than just cutting out actix from dependencies, but it actually what we want to do -- we want to break monolithic primitives crate into smaller but meaningful crates.